src/components/FormLayout/FormLayout.scss

```diff
@@ -24,10 +23,13 @@ $item-min-size: rem(220px);
 
 .Item {
   flex: 1 1 $item-min-size;
-  margin-top: spacing();
   margin-left: spacing(loose);
   max-width: calc(100% - #{spacing(loose)});
 
+  &:not(.First) {
+    margin-top: spacing();
+  }
```

Comment:
Could we do...
```suggestion
  & + .Item {
    margin-top: spacing();
  }
```

Reply:
Will try that! it would be much cleaner, thanks!